src/libraries/System.Speech/src/System.Speech.csproj

```diff
@@ -6,6 +6,15 @@
     <!-- SA1129: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3277 -->
     <NoWarn>$(NoWarn);CS0649;SA1129;CA1847</NoWarn>
     <SetIsTrimmable>false</SetIsTrimmable>
+    <IsPackable>true</IsPackable>
+    <!-- TODO: Enable when package shipped with NET6. -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Comment:
we have shipped a 5.0.0 version of the package.

Reply:
You are correct. It's actually missing from our "nuget-build" mirror feed. Pinged dnceng to add it to it.